tests/test_EscCol.py

```diff
@@ -18,18 +19,38 @@ def test_basic_attributes(self):
 
     def test_modifiying_gene_conversion(self):
         genome = self.species.get_contig(
-            chromosome="Chromosome", gene_conversion_rate=0, gene_conversion_length=1
+            chromosome="Chromosome", gene_conversion_rate=0.1, gene_conversion_length=1
         )
-        assert genome.gene_conversion_rate == 0
+        assert genome.gene_conversion_rate == 0.1
         assert genome.gene_conversion_length == 1
 
+    def test_use_species_gene_conversion(self):
+        genome = self.species.get_contig(
+            chromosome="Chromosome", use_species_gene_conversion=True
+        )
+        assert genome.gene_conversion_rate == 8.9e-11
+        assert genome.gene_conversion_length == 345
+
     def test_unnamed_contig_with_gc(self):
         genome = self.species.get_contig(
-            length=42, gene_conversion_rate=0, gene_conversion_length=1
+            length=42, gene_conversion_rate=0.1, gene_conversion_length=1
         )
-        assert genome.gene_conversion_rate == 0
+        assert genome.gene_conversion_rate == 0.1
         assert genome.gene_conversion_length == 1
 
+    def test_use_species_gene_conversion_unnamed_contig(self):
+        genome = self.species.get_contig(length=42, use_species_gene_conversion=True)
+        assert genome.gene_conversion_rate == 8.9e-11
+        assert genome.gene_conversion_length == 345
+
+    def test_use_species_gene_conversion_unnamed_contig_undefined_gc(self):
```

Comment:
Oh good - I'm glad you're testing this, but it should probably be a test in test_species.py? (see suggestion below)